platform/rust/src/wrapper.cpp

```diff
@@ -0,0 +1,47 @@
+#include "maplibre-native/include/tile_server_options.h"
+#include "maplibre-native/include/map_renderer.h"
+#include "maplibre-native/src/lib.rs.h"
+
+namespace ml {
```

Comment:
I suggest using `mln::rustbridge` as namespace name.

`ml` is Machine Learning for me